plugin/plugin_source.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/cloudquery/plugin-sdk/v4/glob"
 	"github.com/cloudquery/plugin-sdk/v4/message"
+	"github.com/cloudquery/plugin-sdk/v4/scheduler"
```

Comment:
Unsure if it's worth the extra import. This is the only line in the `plugin` package to refer to `scheduler`.

Reply:
We can put this here https://github.com/cloudquery/plugin-sdk/tree/main/helpers

Reply:
`scheduler` already imports `helpers` (DFS uses `helpers.InterfaceSlice()` so it has to be a separate package. `opts` package, `func SchedulerOpts()`?

Reply:
Now I see 
![image](https://github.com/user-attachments/assets/570ff939-faaa-4aa2-a875-026b9040ed83)

Reply:
forgot to `git add`, sorry fixed in https://github.com/cloudquery/plugin-sdk/pull/1900/commits/4d699f5bab90c9770e3f1e2766c39fca2b8b62b3